doc/source/upcoming_changes/towncrier_template.jinja

```diff
@@ -0,0 +1,33 @@
+{%- set title ="Announcement: scikit-image {}".format(versiondata.version) -%} {{ title }} {{ "\n"}}
+{%- set underline = "=" -%} {{ underline * title|length }}
+
+{{ "We're happy to announce the release of scikit-image v{}!".format(versiondata.version) }}
+
+{{ "scikit-image is an image processing library for the scientific Python
+ecosystem that includes algorithms for segmentation, geometric
+transformations, feature detection, registration, color space
+manipulation, analysis, filtering, morphology, and more.\n
+For more information, examples, and documentation, please visit our website:\n
+https://scikit-image.org" }}
+
+{% for section in sections %}
+{% set underline = "-" %} 
+{% if sections[section] %} {% for category, val in definitions.items() if category in sections[section] and category != 'trivial' %}
+
+{{ definitions[category]['name'] }}
+{{ underline * definitions[category]['name']|length }}
+{% if definitions[category]['showcontent'] %} {%- for text, values in sections[section][category]|dictsort(by='value') %} 
+- {{ text }} {% if category != 'process' %} {{ values|sort|join(', ') }} {% endif %} {% endfor %}
+{% else %} 
```

Comment:
```suggestion
- {{ text }} {% if category != 'process' %} {{ values|sort|join(', ') }} {% endif %} {% endfor %}

{% else %} 
```
Seems to need this blank line here, otherwise I get the following in a case I tested with two `doc` and one `bugfix` entry (Note the missing blank line before the start of the "Documentation" section)

```
Bug Fixes
---------
 
- Minor fixups to Hough line transform code and examples  `#5182 <https://github.com/scikit-image/scikit-image/issues/5182>`_    
Documentation
-------------
 
- Prevent integer overflow in EllipseModel  `#5179 <https://github.com/scikit-image/scikit-image/issues/5179>`_   
- update reference link in exposure module  `#5473 <https://github.com/scikit-image/scikit-image/issues/5473>`_
```